File: ComparaNombres/app/src/main/java/com/example/comparanombres/MainActivity.java
package com.example.comparanombres;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private EditText Entrada;
    private TextView Salida;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Entrada =   (EditText)findViewById(R.id.etEntrada);
        Salida  =   (TextView)findViewById(R.id.tvSalida);
    }
    String[] nombres = {"Eduardo","Pedro"};     // -- Arreglo con los nombres a comparar.
    int i = 0;
    boolean c=false;

    public void Btn (View v) {
        //var data1S:String = etxtv.text.toString()
        //String data1 = etEntrada.text.toString()
        String entrada  =   Entrada.getText().toString();
        for(i=0;i<2;i++) {                      //-- Ciclo para comparar la cadena de entrada con las cadenas del arreglo 'nombres'
            if(nombres[i].equals(entrada)) {
                c=true;                         //-- Bandera indica coincidencia
            }
        }
        if(c==false){
            Salida.setText("Diferente");
        }else{
            Salida.setText("Igual");
            c=false;
        }
    }
}

File: Calculator/app/src/main/java/com/example/calculator/MainActivity.kt
package com.example.calculator

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.vieW.View

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnAdd.setOnClickListener(object: View.onClickListener{
            override fun onCLick(v: View) {
                // TODO(reason: "not implemented") //To change body
                var data1S:String = etxData1.text.toString()
                if (data1S.isEmpty()) {
                etxData1.setError("Error")
                }
                else{
                    var data1 = data1S.toInt()
                    var data2S = etxData2.text.toString()
                    var data2 = data2S.toInt()
                    var add = data1 + data2
                    textView6.setText("$add")
                }
            }
        })

        btnSus.setOnClickListener(object: View.onClickListener{
            override fun onCLick(v: View) {
                // TODO(reason: "not implemented") //To change body
                var data1S:String = etxData1.text.toString()
                var data1 = data1S.toInt()
                var data2S = etxData2.text.toString()
                var data2 = data2S.toInt()
                var sus = data1+data2
                textView6.setText("$sus")
            }
        })

        btnMul.setOnClickListener(object: View.onClickListener{
            override fun onCLick(v: View) {
                // TODO(reason: "not implemented") //To change body
                var data1S:String = etxData1.text.toString()
                var data1 = data1S.toInt()
                var data2S = etxData2.text.toString()
                var data2 = data2S.toInt()
                var mul = data1+data2
                textView6.setText("$mul")
            }
        })

        btnDiv.setOnClickListener(object: View.onClickListener{
            override fun onCLick(v: View) {
                // TODO(reason: "not implemented") //To change body
                var data1S:String = etxData1.text.toString()
                var data1 = data1S.toInt()
                var data2S = etxData2.text.toString()
                var data2 = data2S.toInt()
                var div = data1+data2
                textView6.setText("$div")
            }
        })
    }


}
